src/mmw/js/src/data_catalog/templates/resultDetailsCuahsi.html

```diff
@@ -6,33 +6,43 @@
         <h2>
             Site: {{ location }} {{ title }}
         </h2>
-        <p>
-            Source: {{ service_org }} {{ service_title }}
-            <a data-toggle="popover" tabindex="0"
-               data-html="true" data-container="body" role="button"
-               data-content="{{ description }}"
-               data-template="<div class='popover'><div class='pull-right' id='popover-close-button' onclick='closePopover()'><i class='fa fa-times' /></div><div class='popover-content'></div><div class='arrow'></div></div>">
-                <i class="fa fa-info-circle black"></i>
-            </a>
-        </p>
+        <table class="table-data-detail">
+            <tr>
+                <th>Source</th>
+                <td>{{ service_org }} {{ service_title }}
+                    <a data-toggle="popover" tabindex="0"
+                       data-html="true" data-container="body" role="button"
+                       data-content="{{ description }}"
+                       data-template="<div class='popover'><div class='pull-right' id='popover-close-button' onclick='closePopover()'><i class='fa fa-times' /></div><div class='popover-content'></div><div class='arrow'></div></div>">
+                        <i class="fa fa-info-circle black"></i>
+                    </a>
+                </td>
+            </tr>
+            {% if author %}
+                <tr>
+                    <th>Author</th>
+                    <td>
+                        {{ author }}
+                    </td>
+                </tr>
+            {% endif %}
+            <tr>
+                {% if begin_date == end_date %}
+                    <th>Data collected on</th>
+                    <td>{{ begin_date|toDateWithoutTime }}</td>
+                {% else %}
+                    <th>Data collected between</th>
+                    <td>{{ begin_date|toDateWithoutTime }} - {{ end_date|toDateWithoutTime }}</td>
```

Comment:
I realize this is copied over from the previous implementation, but since we're making stylistic improvements, can we replace the ` - ` with a `&ensp;&ndash;&ensp;`?